src/WebJobs.Script/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -50,6 +50,11 @@ public static bool IsAntaresInternalRequest(this HttpRequestMessage request)
             return !request.Headers.Contains(ScriptConstants.AntaresLogIdHeaderName);
         }
 
+        public static bool IsAuthDisabled(this HttpRequestMessage request)
```

Comment:
This is just for naming but maybe `IsBypassAuth()`  or `SkipAuth()` would make more sense as it seems that 'auth disabled' shouldn't be a property of a request. But naming is up to you. You're in this code more than I'm.